rasa/shared/core/domain.py

```diff
@@ -165,7 +165,7 @@ def from_yaml(cls, yaml: Text, original_filename: Text = "") -> "Domain":
 
     @classmethod
     def from_dict(cls, data: Dict) -> "Domain":
```

Comment:
@wochinge, what does this function do? Trying to come up with a docstring.

Reply:
done